### DISCUSSION TITLE
"PictureBook of Secure Cooperation" slides from 2004

### DISCUSSION THREAD

**Discussion by dckc** [Agoric/agoric-sdk#8077] [Show and tell]
If you haven't flipped through these slides (lately), I highly recommend them:

 - [PictureBook of Secure Cooperation](http://erights.org/talks/efun/SecurityPictureBook.pdf) by Marc Stiegler, 2004

<img alt="PictureBook of Secure Cooperation" src="https://github.com/Agoric/agoric-sdk/assets/150986/7fc6b982-a607-4986-aca2-cc5deeab1c90" >

Secure Cooperation is what Agoric is all about:

> The same design principles reappear at each layer of our stack: Enable **more cooperation with less risk**, by limiting, subdividing, and composing fine-grain electronic rights. -- [Agoric’s Composable Smart Contract Framework Reaches Mainnet\-1 Milestone](https://agoric.com/blog/announcements/agoric-composable-smart-contract-framework-reaches-mainnet-1-milestone) Oct 2022

@anilhelvaci , @Jorge-Lopes have you seen it? Did you come across it while preparing your [lectures](https://github.com/Chainboard-Academy/agoric-lecture-content/)?

These slides are in the [talks section of awesome-ocap](https://github.com/dckc/awesome-ocap#presentations-talks-slides-and-videos); you might like the others there too.

cc @gibson042 @iomekam @erights @hielo777

**Comment by Jorge-Lopes** [comment]
Hello @dckc 

I remember seeing this before, but I think it was in some previous office hours section.
Although, I don't think we use it for the lecture's content. But Anil can confirm it.

Nevertheless, thank you very much for sharing it. 
I will definitely go through it again.

**Comment by dckc** [comment]
Another gem by Stiegler:

 - [The Lazy Programmer's Guide to Secure Computing](https://www.youtube.com/watch?v=eL5o4PFuxTY) 2010 Google Tech Talk
